Kitodo/src/test/java/org/kitodo/selenium/MetadataST.java

```diff
@@ -330,6 +330,29 @@ public void createStructureElementTest() throws Exception {
         assertEquals(structureType, firstChild.getText(), "Added structure element has wrong type!");
     }
 
+    /**
+     * Verifies that turning the "show physical page number below thumbnail switch" on in the user settings
+     * results in thumbnail banner being displayed in the gallery of the metadata editor.
+     */
+    @Test
+    public void showPhysicalPageNumberBelowThumbnailTest() throws Exception {
+        login("kowal");
+       
+        // open the metadata editor
+        Pages.getProcessesPage().goTo().editMetadata(MockDatabase.MEDIA_RENAMING_TEST_PROCESS_TITLE);
+        
+        // verify that physical page number is not shown below thumbnail by default
+        assertEquals(0, Browser.getDriver().findElements(By.cssSelector(".thumbnail-banner")).size());
+
+        // change user setting
+        Pages.getMetadataEditorPage().closeEditor();
+        Pages.getUserEditPage().toggleShowPhysicalPageNumberBelowThumbnail();
+        Pages.getProcessesPage().goTo().editMetadata(MockDatabase.MEDIA_RENAMING_TEST_PROCESS_TITLE);
+
+        // verify physical page number is now shown below thumbnail
+        assertTrue(0 < Browser.getDriver().findElements(By.cssSelector(".thumbnail-banner")).size());
```

Comment:
Could be simplified to
```suggestion
        assertFalse(Browser.getDriver().findElements(By.cssSelector(".thumbnail-banner")).isEmpty());
```